judgels-backends/jophiel/jophiel-app/src/main/java/judgels/jophiel/user/account/UserRegisterer.java

```diff
@@ -63,4 +63,10 @@ public void activate(String emailCode) {
             throw new NotFoundException();
         }
     }
+
+    public void resendActivationEmail(User user) {
+        Optional<String> maybeEmailCode = userRegistrationEmailStore.getEmailCode(user.getJid());
```

Comment:
Can we check that the user is not activated yet. Otherwise I can spam anyone with activation email using this.

Reply:
updated in `UserAccountResource`

https://github.com/ia-toki/judgels/pull/209/commits/5943d5d4faa2efcc89d97dc0c98125f80c3e3942#diff-3458bd16b4c06c39966a4c22e21b5e6aR57